src/helpers/spaces.ts

```diff
@@ -56,6 +56,10 @@ type Metadata = {
   pluginNames: string[];
 };
 
+function isTurbo(turbo: number, turboExpiration: number): boolean {
+  return turbo === 1 || turboExpiration > Date.now() / 1e3;
```

Comment:
Maybe we can just use `turboExpiration > Date.now() / 1e3` ? i remember at some point, the plan was to deprecate `turbo` right?

Reply:
Yes, but until we do, we still need this for the transition period, as we're filling up the schnaps db with data